web/crux/src/grpc/google/protobuf/timestamp.ts

```diff
@@ -111,16 +115,14 @@ export interface Timestamp {
 
 export const GOOGLE_PROTOBUF_PACKAGE_NAME = 'google.protobuf'
 
-function createBaseTimestamp(): Timestamp {
-  return { seconds: 0, nanos: 0 }
-}
+const baseTimestamp: object = { seconds: 0, nanos: 0 }
```

Comment:
Again?

Reply:
I'll check this.

Reply:
The `alpine-proto:3.16` image seems to be outdated. After building the image locally it generates the same timestamp.ts as in the repo.